tests/integration/api_v3/http/blockchainApps.test.js

```diff
@@ -39,75 +37,67 @@ describe('Blockchain apps API', () => {
 		curChainID = response.data.chainID;
 	});
 
-	it('retrieves list of all blockchain applications', async () => {
+	it('should return list of all blockchain applications', async () => {
 		const response = await api.get(endpoint);
-		expect(response).toMap(goodRequestSchema);
-		expect(response.data).toBeInstanceOf(Array);
 		expect(response.data.length).toBeGreaterThanOrEqual(1);
 		expect(response.data.length).toBeLessThanOrEqual(10);
-		response.data.map(blockchainApp => expect(blockchainApp).toMap(blockchainAppSchema));
-		expect(response.meta).toMap(metaSchema);
+		expect(response).toMap(blockchainAppsSchema);
 	});
 
-	it('retrieves list of all blockchain applications with limit=10', async () => {
+	it('should return list of all blockchain applications when called with limit=10', async () => {
 		const response = await api.get(`${endpoint}?limit=10`);
-		expect(response).toMap(goodRequestSchema);
-		expect(response.data).toBeInstanceOf(Array);
 		expect(response.data.length).toBeGreaterThanOrEqual(1);
 		expect(response.data.length).toBeLessThanOrEqual(10);
-		response.data.map(blockchainApp => expect(blockchainApp).toMap(blockchainAppSchema));
-		expect(response.meta).toMap(metaSchema);
+		expect(response).toMap(blockchainAppsSchema);
 	});
 
-	it('retrieves list of all blockchain applications with limit=10 and offset=1', async () => {
+	it('should return list of all blockchain applications when called with limit=10 and offset=1', async () => {
 		const response = await api.get(`${endpoint}?limit=10&offset=1`);
-		expect(response).toMap(goodRequestSchema);
-		expect(response.data).toBeInstanceOf(Array);
 		expect(response.data.length).toBeGreaterThanOrEqual(0);
 		expect(response.data.length).toBeLessThanOrEqual(10);
-		response.data.map(blockchainApp => expect(blockchainApp).toMap(blockchainAppSchema));
-		expect(response.meta).toMap(metaSchema);
+		expect(response).toMap(blockchainAppsSchema);
 	});
 
-	it('retrieves blockchain application by chainID', async () => {
+	it('should return blockchain application when called by valid chainName', async () => {
+		const response = await api.get(`${endpoint}?chainName=enevti`);
+		expect(response.data.length).toEqual(1);
+		expect(response).toMap(blockchainAppsSchema);
```

Comment:
Its better to check data.length after checking schema since schema validates if data passed in response is an array 
```suggestion
		expect(response).toMap(blockchainAppsSchema);
		expect(response.data.length).toEqual(1);
```